src/CalculatorUITestFramework/NavigationMenu.cs

```diff
@@ -0,0 +1,105 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace CalculatorUITestFramework
+{
+    public enum CalculatorMode
+    {
+        StandardCalculator,
+        ScientificCalculator,
+        ProgrammerCalculator,
+        DateCalculator,
+        Currency,
+        Volume,
+        Length,
+        Weight,
+        Temperature,
+        Energy,
+        Area,
+        Speed,
+        Time,
+        Power,
+        Data,
+        Pressure,
+        Angle
+    }
+    public class NavigationMenu : ApplicationBase
```

Comment:
If this class is not meant to be instantiated, make it a static class.

Same feedback for the rest of the framework classes.

Reply:
It gets instantiated by the StandardCalculatorPage.cs. My thought was to have the StandardCalculatorPage.cs be the class that brought all of the components together to be used. My thought was that it would be clearer what was available to each mode this way.